src/component/page/gjennomforing-page/gjennomforing-liste/DeltakerListe.tsx

```diff
@@ -0,0 +1,52 @@
+import { Alert } from '@navikt/ds-react'
+import React, { useMemo } from 'react'
+
+import globalStyles from '../../../../globals.module.scss'
+import { sortAlphabeticAsc } from '../../../../utils/sortering-utils'
+import styles from './DeltakerListe.module.scss'
+import { GjennomforingListePanel } from './GjennomforingListePanel'
+import { TiltakSection } from './TiltakSection'
+import { Deltakerliste } from '../../../../api/data/deltaker'
+import { finnDeltakerlister, finnUnikeDeltakerlister } from '../../../../utils/deltakerliste-utils'
+
+interface DeltakerListeProps {
+	deltakerliste: Deltakerliste[];
+}
+
+export const DeltakerListe = (props: DeltakerListeProps): React.ReactElement<DeltakerListeProps> => {
+	const unikeDeltakerlister = useMemo<Deltakerliste[]>(() => {
```

Comment:
Jeg forstår skifte fra gjennomføring til deltakerliste, men skal vi gå helt bort i fra ordet tiltak også? Jeg synes dette er litt forvirrende nå, for disse `unikeDeltakerlister` er vel egentlig unike tiltak, slik det var før, for å lage overskrifter og gruppere deltakerlister under.

Reply:
Jeg kan godt endre tilbake til tiltak her hvis det er mer riktig?

Reply:
Jeg tror i allefall tiltak og tiltakstyper / tiltaksvarianter er noe vi vil fortsatt ha et forhold til, og at det er mer riktig å bruke det her. 

Alternativt så kan man i allefall trekke frem at det er unike typer av lister det er snakk om.

Reply:
Det vi egentlig gjør her er å finne unike tiltakstyper, så jeg endrer i henhold til det :) Ser at jeg også kan rydde litt i metoden så den ikke returnerer en helt deltakerliste.